netci.groovy

```diff
@@ -13,7 +13,8 @@ def addArchival(def job) {
   // Grab these logs from all of those locations.
   [ "", "source-build/", "tarball-output/"].each { logRoot ->
     archivalSettings.addFiles("${logRoot}bin/logs/*")
-    archivalSettings.addFiles("${logRoot}bin/prebuilt-report/*")
+    archivalSettings.addFiles("${logRoot}bin/prebuilt-report/**/*")
```

Comment:
Does this still pick up loose files in `prebuilt-report`?  The XML summaries will still be going in there, right?

Reply:
It should--`**` is meant to match no/current directory as well as all recursive subdirectories.

Reply:
Testing this with https://ci.dot.net/job/dotnet_source-build/job/release_2.1/job/GenPRTest/job/Ubuntu16.04_Release_prtest/1/